ember-intl/package.json

```diff
@@ -17,21 +17,29 @@
   },
   "license": "MIT",
   "author": "jason.mitchell.w@gmail.com",
+  "typesVersions": {
+    "*": {
+      "*": [
+        "declarations/*",
+        "declarations/*/index.d.ts"
```

Comment:
This is likely wrong causing the issues. See https://github.com/ember-cli/ember-cli/blob/fa8599058bde3c745a2b1156076955ca3543c857/blueprints/addon/index.js#L101-L107